sqlscripts/local/0001-jaeger-index.sql

```diff
@@ -4,8 +4,9 @@ CREATE TABLE IF NOT EXISTS %s (
      service LowCardinality(String) CODEC(ZSTD(1)),
      operation LowCardinality(String) CODEC(ZSTD(1)),
      durationUs UInt64 CODEC(ZSTD(1)),
-     tags Array(String) CODEC(ZSTD(1)),
-     INDEX idx_tags tags TYPE bloom_filter(0.01) GRANULARITY 64,
+     tag_keys Array(String) CODEC(ZSTD(1)),
+     tag_values Array(String) CODEC(ZSTD(1)),
```

Comment:
I think it'd be better to use nested structure instead of parallel arrays.  

Something like

```
tags Nested {
  key String CODEC(ZSTD(1)),
  value String CODEC(ZSTD(1)),
}
```

https://clickhouse.tech/docs/ru/sql-reference/data-types/nested-data-structures/nested/ 

And if the number of unique key values is relatively small (< 10 000), consider using `LowCardinality` data type

https://clickhouse.tech/docs/en/sql-reference/data-types/lowcardinality/